ghcide/src/Development/IDE/Plugin/HLS/GhcIde.hs

```diff
@@ -28,6 +29,7 @@ descriptors =
     CodeAction.fillHolePluginDescriptor "ghcide-code-actions-fill-holes",
     Completions.descriptor "ghcide-completions",
     TypeLenses.descriptor "ghcide-type-lenses",
+    SelectionRange.descriptor "ghcide-selection-range",
```

Comment:
Given that this has been set up as a new plugin, I would prefer if the plugin lived in its own package and ghcide didn't depend on it. This keeps ghcide small and makes for easier GHC upgrades (as mentioned in https://github.com/haskell/haskell-language-server/issues/212#issuecomment-1003618821)

Reply:
Ok, I will move this plugin to its own package. By the way, if I understand correctly, plugins in ghcide is somewhat 'legacy' and should be moved to their own packages eventually in the future if possible, right?

Reply:
yeah, that is the long term plan, we should note it in the contributors guide